docs/src/appendix/scarb-toml.md

```diff
@@ -158,14 +185,17 @@ The `starknet-contract` target allows to build the package as a Starknet Contrac
 ```
 
 #### `sierra`
+
 See more about [Sierra contract class generation](https://docs.swmansion.com/scarb/docs/extensions/starknet/contract-target.html#sierra-contract-class-generation) in Scarb documentation.
+
 ```toml
 [[target.starknet-contract]]
 sierra = true
 ```
 
 #### `casm`
 
+Enabling [`casm = true`](https://github.com/foundry-rs/starknet-foundry/releases/tag/v0.23.0) in Scarb.toml causes unnecessary overhead and should be disabled unless required by other tools. Starknet Foundry compiles Sierra to CASM separately, resulting in redundant processing. This duplicates CASM generation, significantly impacting performance, especially for large Sierra programs.
```

Comment:
@NueloSE changes mentioned in https://github.com/foundry-rs/starknet-foundry/pull/2619#discussion_r1819023130 are still not applied. Please remove the link.